tests/framework/artifacts.py

```diff
@@ -468,3 +470,58 @@ def disks(self):
     def ssh_key(self):
         """Return the ssh key file path."""
         return self._ssh_key
+
+    @property
+    def net_ifaces(self):
+        """Return the ssh key file path."""
+        return self._net_ifaces
+
+
+# Default configuration values for network interfaces.
+DEFAULT_HOST_IP = "192.168.0.1"
```

Comment:
Do we want to have default IP addresses? Sounds like a recipe for duplicate IP addresses when writing tests.

Reply:
Two uVMs can use the same network configuration without any problem since they all live in different net namespaces. To make them communicate the multiple uVM tests would need to do some plumbing work.

Reply:
Right, but on the host side there may be collisions.